lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -263,7 +256,7 @@ export default function ContentSelector({
           authToken={authToken}
           listFilesApi={blackboardListFilesApi}
           onCancel={cancelDialog}
-          onSelectFile={selectFileAsURL}
+          onSelectFile={selectBlackBoardFile}
```

Comment:
These are new, we were not using the nice formatting for these data sources for some reason :shrug: